WireAPI/Sources/WireAPI/APIs/ConversationsAPI/ConversationsAPIV0.swift

```diff
@@ -54,15 +57,29 @@ class ConversationsAPIV0: ConversationsAPI, VersionedAPI {
             )
             let response = try await self.httpClient.executeRequest(request)
 
-            return try ResponseParser()
-                .success(code: 200, type: PaginatedConversationIDsV0.self)
-                .parse(response)
+            return try self.decodeConversationIdentifiers(from: response)
+        }
+    }
+
+    private func decodeConversationIdentifiers(from response: HTTPResponse) throws -> PayloadPager<QualifiedID>.Page {
+        guard let data = response.payload else {
+            throw ResponseParserError.missingPayload
+        }
+
+        let decoder = JSONDecoder.defaultDecoder
+
+        switch response.code {
+        case 200..<400:
+            let payload = try decoder.decode(PaginatedConversationIDsV0.self, from: data)
+            return payload.toAPIModel(domain: backendDomain)
+        default:
+            throw try decoder.decode(FailureResponse.self, from: data)
         }
     }
 
 }
 
-private struct PaginatedConversationIDsV0: Decodable, ToAPIModelConvertible {
+private struct PaginatedConversationIDsV0: Decodable {
```

Comment:
It was not possible to use `ToAPIModelConvertible`, because of the additional required value. Then I implemented the decoding manually again.
Can we find a better design?

Reply:
thanks, I choose a different implementation where `ToAPIModelConvertible` works.